app/Http/Repositories/MailChimpRepository.php

```diff
@@ -307,6 +307,27 @@ public function sendCampaign(string $campaignId) : bool
         }
     }
 
+    /**
+     * @see https://developer.mailchimp.com/documentation/mailchimp/reference/campaigns/#action-post_campaigns_campaign_id_actions_test
+     */
+    public function sendTestNewsletter(string $email, string $campaignId): string {
+      $mailChimpApiArguments = [
+        'test_emails' => ["{$email}"],
+        'send_type' => "html"
+      ];
+      try {
+        $this->post("campaigns/{$campaignId}/actions/test", $mailChimpApiArguments);
+        return "This worked.";
```

Comment:
This bad boy should probably return a `boolean` instead of a string. This MailChimp action when successful doesn't look like it returns anything, so it basically succeeds or doesn't. In this case, `sendTestNewsletter`, if it works, should return a `true`.

Reply:
This isn't part of any change request but I totally never mentioned that we follow the PSR-2 PHP style conventions https://www.php-fig.org/psr/psr-2/ . On some repositories we have codeclimate setup so that you might get some annoying flags about brackets being on this-or-that line. No biggie, but it just dawned on me so I thought I'd mention it.

Reply:
Thanks for calling that out! I missed it while clearing out comments.

Reply:
This has been changed to return `true`.